subsys/nrf_security/src/drivers/cracen/silexpk/src/ed448.c

```diff
@@ -9,6 +9,7 @@
 #include <silexpk/core.h>
 #include <silexpk/ed448.h>
 #include <silexpk/ec_curves.h>
+#include <cracen/math_helpers.h>
```

Comment:
did you intend to use the helper functions in this file as well?

Reply:
No, removed it